distributed_training/utils/gradient_averager.py

```diff
@@ -336,7 +275,6 @@ def step(
         allow_retries: bool = True,
         require_trigger: bool = False,
         wait: bool = True,
-        peerids_to_uids: dict = {},
```

Comment:
@mikkelmedm how come we removed this given that we need to provide it in line 327?